.travis.yml

```diff
@@ -7,12 +7,8 @@ python:
 
 env:
   matrix:
-    - DJANGO_VERSION='>=1.10,<1.11' POSTGRES_VERSION=9.5
-    - DJANGO_VERSION='>=1.11,<1.12' POSTGRES_VERSION=9.5
     - DJANGO_VERSION='>=2.2,<2.3' POSTGRES_VERSION=9.5
-    - DJANGO_VERSION='>=1.10,<1.11' POSTGRES_VERSION=10
-    - DJANGO_VERSION='>=1.11,<1.12' POSTGRES_VERSION=10
-    - DJANGO_VERSION='>=2.2,<2.3' POSTGRES_VERSION=10
+    - DJANGO_VERSION='>=2.0,<3.3' POSTGRES_VERSION=10
```

Comment:
I think we need at least one more line here. We have installs of Django 2.2 and (now) 3.2, so we should test them at least. It's also a good idea to test the next version of Django to see if one exists — that way we can catch errors in future versions early. So, we can have:

```
DJANGO_VERSION='>=2.2,<2.3' POSTGRES_VERSION=10
DJANGO_VERSION='>=3.2,<3.3' POSTGRES_VERSION=10
DJANGO_VERSION='>3.2' POSTGRES_VERSION=10
```

That gives us: 
1. 2.2.X
2. 3.2.X
3. >3.2 (Django 4, but that's not out until December this year)